announcement/announcement-api/api/src/bundle/announcement_eu.properties

```diff
@@ -196,6 +196,8 @@ revise.required.star=Ezinbesteko itemak izartxo batekin markatuak daude\: <span
 revise.update=Eguneratu inprimakia eta sakatu botoi egokia.
 revise.announce=Oharra
 revise.announcement.body=Gorputza
+revise.highlight = <strong>Erakutsi iragarki hau.</strong> Iragarpen zerrendan nabarmenagoa izango da.
+revise.star.title = Iragarki hau agertu da
```

Comment:
@danielmerino Please help us! haha

Reply:
@JuanDavid102 

This is going to be....

revise.highlight = <strong>Nabarmendu iragarki hau</strong>. Jarri iragarkien zerrendan gehiago ikusteko moduan.
revise.star.title = Iragarki hau nabarmendu da